core/src/test/kotlin/org/stellar/anchor/util/LogTest.kt

```diff
@@ -7,30 +7,25 @@ import io.mockk.impl.annotations.MockK
 import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.Assertions.*
 import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.TestInstance
 import org.slf4j.Logger
 import org.stellar.anchor.config.AppConfig
 import org.stellar.anchor.config.PII
+import org.stellar.anchor.lockAndMockStatic
 import org.stellar.anchor.util.Log.shorter
 import org.stellar.sdk.Network.TESTNET
 
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Disabled
```

Comment:
Yes, this is interfering with all other tests with Logger calls. When we mock them, some other tests may fail. I temporarily disabled this and added a ticket to re-enable them. Probably need to use TestSuite so that they won't run with other tests.